open_lm/model.py

```diff
@@ -299,7 +309,7 @@ def create_params(args):
         weight_tying=cfg["weight_tying"],
         norm_type=get_norm_class(args.model_norm),
         apply_qk_norm=args.qk_norm,
-        rotary_old=args.rotary_old,
+        positional_embedding_type=cfg["positional_embedding_type"],
```

Comment:
old configs don't have a "positional_embedding_type", can we set the default to be "rotary"?

Reply:
old configs might also match old models that use head_rotary so I thought it'd be better to fail explicitly. Tell me if you still want me to put rotary as default.

Reply:
nice!